bindings/blst_aux.h

```diff
@@ -99,4 +99,10 @@ size_t blst_p1_affine_sizeof();
 size_t blst_p2_sizeof();
 size_t blst_p2_affine_sizeof();
 size_t blst_fp12_sizeof();
+
+/*
+ * SHA-256 hash function.
+ */
+void blst_sha256(unsigned char md[32], const void *msg, size_t len);
```

Comment:
Not that it actually matters in the context, but blst.h follows a specific convention for naming arguments that reflects so called typemaps in blst.swg. For this reason I'll change this line as I commit.

BTW, how would `byte *msg` look at your side? I mean `void*` is a convenient way to avoid specific type warnings, but at the same type it masks potentially meaningful programming mistakes. For example passing pointer to an integer might work in specific context, but formally speaking it's not exactly legit.

Reply:
A `byte *msg` would be fine on our end. Our input is `uint8_t*` (`byte*`) so it works well.